external/llvm-project/mlir/lib/Dialect/Tosa/Transforms/TosaPartition.cpp

```diff
@@ -88,26 +89,53 @@ bool isElementwiseOp(Operation *op) {
   // clang-format on
 }
 
+bool isFusibleOp(Operation *op) {
+  return isElementwiseOp(op) || isa<tosa::TransposeOp, tosa::ReshapeOp>(op);
+}
+
+bool isZeroAttribute(Attribute value) {
```

Comment:
Moved out because they don't need to be member functions.

Reply:
In which case they get a `static`

Reply:
Yes.  I just saw the anonymous-namespace part of the coding-standards document.